test/e2e/framework/metrics/interesting_metrics.go

```diff
@@ -18,9 +18,7 @@ package metrics
 
 var interestingAPIServerMetrics = []string{
 	"apiserver_request_total",
-	// TODO(krzysied): apiserver_request_latencies_summary is a deprecated metric.
-	// It should be replaced with new metric.
-	"apiserver_request_latencies_summary",
+	"apiserver_request_latency_seconds",
```

Comment:
This change seems good for me because of
`apiserver_request_latencies_summary -> apiserver_request_latency_seconds` as https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.14.md#deprecated-metrics